metadata.rb

```diff
@@ -7,7 +7,7 @@
 source_url       'https://github.com/sous-chefs/mariadb'
 issues_url       'https://github.com/sous-chefs/mariadb/issues'
 chef_version     '>= 13'
-version          '2.0.0'
+version          '2.0.3'
```

Comment:
Please revert, we prefer to control the version numbers ourselves :)

Reply:
sure thing =)

Reply:
how about the changelog?

Reply:
In the last commit I left the version reference untouched in the CHANGELOG.md with the changes made.